app/src/features/admin/users/AddSystemUsersForm.tsx

```diff
@@ -80,6 +88,30 @@ const AddSystemUsersForm: React.FC<AddSystemUsersFormProps> = (props) => {
                             }}
                           />
                         </Box>
+                        <Box pl={1} width={220}>
```

Comment:
Seem to be missing padding on the right:

![image](https://github.com/bcgov/biohubbc/assets/18599267/199a2df8-c786-4021-9a51-e86164991be2)

Reply:
Yeah this is super temporary, just to make some testing easier. All of these manual user controls are going to be drastically changed in the next few sprints